### PR TITLE
Fix Typo on quat:nlerp shortest path branch variable

### PR DESCRIPTION
**Pull request by Kuranes**
Typo Bug spotted by @stephomi

### CODE REVIEW

**Comment by cedricpinson** [comment]
At the same time could you add a test?

File: sources/osg/Quat.js [hunk 1]
```diff
@@ -101,17 +101,25 @@ define( [
 
 
         // http://physicsforgames.blogspot.fr/2010/02/quaternions.html
+        // called quatBlend
+        //  (  quaternion nlerp is supposed to be commutative, constant velocity, torque minimal )
```
Comment (cedricpinson) [inline sources/osg/Quat.js]: http://number-none.com/product/Understanding%20Slerp,%20Then%20Not%20Using%20It/ tells is not commutive

**Comment by Kuranes** [comment]
added test on nlerp and slerp.

See comment about reusing Vec4 code directly ?
(not inheriting, but reusing code for smaller js files)

**Comment by cedricpinson** [comment]
Agree you can add reference to Vec4 method no need for duplicate code

**Comment by Kuranes** [comment]
Done.

**Pull request merged**